content/3.Joining Corda Network/5.Offboarding Workflow.md

```diff
@@ -0,0 +1,22 @@
+If you foresee that one or some of your Corda Network network certificate(s) will not be used permanently or will not re-join in the long-run, you may submit a request to revoke the certificate.
```

Comment:
we should refer to them as X.509 certificates throughout

Reply:
updated